### PR TITLE
Initial BE skeleton setup

### PR DESCRIPTION
**Pull request by dimitrov93**
GET point to /home

### CODE REVIEW

**Comment by dimitrov93** [comment]
Initial setup

File: .idea/workspace.xml
Comment (VladoKat) [inline .idea/workspace.xml:1]: We should ignore some of .idea's contents. Please, refer to this article by JetBrains -> https://git-scm.com/book/en/v2/Getting-Started-The-Command-Line. They have mentioned a .gitignore file to use https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore

File: jewellery-inventory/HELP.md
Comment (VladoKat) [inline jewellery-inventory/HELP.md:1]: Do we need this file? I imagine it's generated automatically.

File: jewellery-inventory/.vscode/launch.json
Comment (VladoKat) [inline jewellery-inventory/.vscode/launch.json:1]: I think we don't need .vscode configuration in this project if we are using Intellij for the backend. Should we add it to the .gitignore file as well?

File: jewellery-inventory/target/classes/application.properties
Comment (VladoKat) [inline jewellery-inventory/target/classes/application.properties:1]: We should ignore the target folder as well.  We don't want to include the built service in Github. -> https://github.com/github/gitignore/blob/main/Maven.gitignore

**Review by VladoKat — CHANGES_REQUESTED** [review]
According to this PR, there are no project files added apart from the .idea, .vscode and target folder. As I've mentioned, we should actually ignore those folders (or parts of them) because otherwise they would pop up in every PR. Another thing is we would want to merge to the test branch first. By the way, let's have the package be `com.jewellery.inventory`.

**Pull request closed without merging**